src/scheduler/TaskScheduler.php

```diff
@@ -33,12 +33,12 @@
 class TaskScheduler{
 	private bool $enabled = true;
 
-	/** @phpstan-var ReversePriorityQueue<int, TaskHandler> */
+	/** @phpstan-var ReversePriorityQueue<int, TaskHandler<Task>> */
 	protected ReversePriorityQueue $queue;
 
 	/**
-	 * @var ObjectSet|TaskHandler[]
-	 * @phpstan-var ObjectSet<TaskHandler>
+	 * @var ObjectSet|TaskHandler<Task>[]
```

Comment:
should this be added to `@var`?

Reply:
Why shouldn't it?

Reply:
I thought generics should only go to `@phpstan-var`

Reply:
Yeah, @var is meant to be PSR-5 compliant so that tools can understand it
Generics should be confined to @phpstan-var